packages/player-react/src/index.tsx

```diff
@@ -114,6 +129,45 @@ export function Player({
     }
   };
 
+  const handlePlayerReady = (event: Event) => {
+    const player = (event as CustomEvent).detail;
+    if (player) {
+      onPlayerReady(player);
+    }
+  };
+
+  const handlePlayerResize = useCallback(
```

Comment:
I had a lot of comments on this function so I just ended up rewriting it
- renamed `entry` to `firstEntry` to make it clearer what the array destructuring does. I didn't know so I had to look it up
- removed a leftover console log
- fixed the nested if (please try to not do nested ifs, if you see some in your code, try to find a way to get rid of them, there are definitely cases where it doesn't make sense but in most cases it improves readability a lot)
- removed `?.` on `onPlayerResize` which should be fine as you're assigning a default 
```tsx
  const handlePlayerResize = useCallback(
    (entries: ResizeObserverEntry[]) => {
      const [firstEntry] = entries;
      if (!firstEntry || !wrapperRef.current) {
        return;
      }

      const newRect = firstEntry.contentRect;
      const sameWidth = newRect.width === lastRect.current.width;
      const sameHeight = newRect.height === lastRect.current.height;
      if (lastRect.current && sameWidth && sameHeight) {
        return;
      }

      lastRect.current = newRect;
      onPlayerResize(newRect);
    },
    [onPlayerResize],
  );
```